pkgs/misc/tmux-plugins/default.nix

```diff
@@ -479,6 +479,35 @@ in rec {
     };
   };
 
+  pass = mkTmuxPlugin {
+    pluginName = "pass";
+    version = "unstable-2020-02-28";
```

Comment:
I think you should prefix it with `0-unstable`
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
Done ✔️